src/odemis/gui/cont/correlation.py

```
@@ -135,7 +134,7 @@ def __init__(self, tab_data, panel, tab, viewports):
             vp.canvas.Bind(wx.EVT_LEFT_DOWN, self.on_mouse_down)
 
         # localization tab
-        self.localization_tab: LocalizationTab = None
+        self.localization_tab: "LocalizationTab" = None
```

Ruff (0.15.6)

[error] Undefined name `LocalizationTab`

(F821)